### PR TITLE
Update java.md

### PR DESCRIPTION
**Pull request by czubair**

### CODE REVIEW

File: _includes/releases/notes/java.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+- For tutorials, samples, quick starts and other documentation, visit [Azure for Java Developers](https://docs.microsoft.com/java/azure/).
+- For build reports on code quality, test coverage, etc, visit [Azure Java SDK](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/index.html).
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose).
+- Check [previous questions](https://stackoverflow.com/questions/tagged/azure-java-sdk) or ask new ones on StackOverflow using `azure-java-sdk` tag.
+
+## Latest Releases
+
+View all the latest versions of Java packages [here][java-latest-releases].
+
+{% include refs.md %}
```
Comment (weshaggard) [inline _includes/releases/notes/java.md:17]: This line should stay at the bottom of the file.

Comment (vcolin7) [inline _includes/releases/notes/java.md:17]:
```suggestion
```

File: _includes/releases/notes/java.md [hunk 2]
```diff
@@ -25,21 +43,3 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
 ## Feedback
 
 If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/azure/azure-sdk-for-java/issues).
```
Comment (vcolin7) [inline _includes/releases/notes/java.md:45]:
```suggestion
If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/azure/azure-sdk-for-java/issues).

{% include refs.md %}
```

**Review by vcolin7 — COMMENTED** [review]
Added suggestions to address what Wes said.

File: _includes/releases/notes/java.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+- For build reports on code quality, test coverage, etc, visit [Azure Java SDK](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/index.html).
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose).
+- Check [previous questions](https://stackoverflow.com/questions/tagged/azure-java-sdk) or ask new ones on StackOverflow using `azure-java-sdk` tag.
+
+## Latest Releases
+
+View all the latest versions of Java packages [here][java-latest-releases].
+
+{% include refs.md %}
+
```
Comment (vcolin7) [inline _includes/releases/notes/java.md:18]:
```suggestion
```

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**